mmcv/image/colorspace.py

```diff
@@ -73,13 +74,124 @@ def gray2rgb(img):
         img (ndarray): The input image.
 
     Returns:
-        ndarray: The converted BGR image.
+        ndarray: The converted RGB image.
     """
     img = img[..., None] if img.ndim == 2 else img
     out_img = cv2.cvtColor(img, cv2.COLOR_GRAY2RGB)
     return out_img
 
 
+def rgb2ycbcr(img, return_y=False):
+    """Convert a RGB image to YCbCr image.
+
+    This function produces the same results as Matlab's rgb2ycbcr function.
+
+    Args:
+        img (ndarray): The input image. It accepts:
+            1. np.uint8 type with range [0, 255];
+            2. np.float32 type with range [0, 1].
+        return_y (bool): Whether to only return Y channel. Default: False.
+
+    Returns:
+        ndarray: The converted YCbCr image. The output image has the same type
+            and range as input image.
+    """
+    img_type = img.dtype
+    img = img.copy().astype(np.float32)
+    if img_type == np.float32:
+        pass
+    elif img_type == np.uint8:
+        img /= 255.
+    else:
+        raise TypeError('The img type should be np.float32 or np.uint8, '
+                        f'but got {img_type}')
```

Comment:
The above logic seems to be shared by multiple methods. You may create a util method.